usr/libexec/setup-etc-subvol

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -e
+#
+# Copyright (c) 2019-2025 SUSE Linux GmbH, Nuernberg, Germany
+#
+# Create nested /etc subvolume and add it to fstab.
+
+# Already there?
+if [ -e /etc/fstab ] && [ -n "$(awk '$2 == "/etc"' /etc/fstab)" ]; then
+	exit 1
```

Comment:
Maybe print a messaage in case some poor soul runs into this and gets confused?

Reply:
Done